packages/nimble-components/docs/internal.mdx

```diff
@@ -1,8 +0,0 @@
-import { Meta } from '@storybook/addon-docs';
```

Comment:
If we go with hiding these only on nimble.ni.dev then maybe we keep this around for devs. Maybe even add a patterns doc as well that contains the same note.